app/manifest.json

```diff
@@ -59,7 +59,10 @@
     "unlimitedStorage",
     "clipboardWrite",
     "http://localhost:8545/",
-    "https://*.infura.io/"
+    "https://*.infura.io/",
+    "activeTab",
+    "webRequest",
+    "*://*.eth/"
```

Comment:
This seems to appear in permissions dialog for Chrome as "eth websites" without an indication it's talking about a TLD. Leaving the user wondering "What is an eth website?"